scripts/build-rollup/plugins.js

```diff
@@ -0,0 +1,32 @@
+const babel = require('rollup-plugin-babel');
+const commonjs = require('rollup-plugin-commonjs');
+const closure = require('rollup-plugin-closure-compiler-js');
+
+// Temporary: If rollup enters, babelrc options need not be set to false
+module.exports = [
+  babel({
+    babelrc: false,
+    presets: [
```

Comment:
can we a single root babelrc like before? there are three babel configs in the PR, is there an advantage to being so specific with them?

Reply:
Its temporary.

1. For existing babel driven build: needs to transform import statements placed inside src/ for convenience
2. For rollup build: module false
3. For examples since I had placed one inside src/

It'll get cleaned up once we settle with the build process.